src/components/UI/Dialog/index.css

```diff
@@ -0,0 +1,91 @@
+.Dialog {
+  padding: 0;
+  text-align: left;
+
+  --dialog-margin-horizontal: 32px;
+  --dialog-margin-vertical: 24px;
+  --dialog-gap: 24px;
+}
+
+.Dialog__element {
```

Comment:
Anything against `camelCase`? ahahah
we use camel case on the other variables and CSS classnames would be good to be consistent :)